docs/usage/filters/interactive-output-filtering.md

```diff
@@ -0,0 +1,32 @@
+---
```

Comment:
Can we move it under `docs/usage`? And renambe `Filters` to `Event filters`, to avoid confusion? These two filters are totally different ones, so I think that would improve understanding of our features 😄

Reply:
Done